content/en/docs/tasks/debug-application-cluster/audit.md

```diff
@@ -579,7 +577,7 @@ A second class of rules tracks resources being created or destroyed, including:
 
 The final class of rules simply displays any Audit Event received by Falco. This rule is disabled by default, as it can be quite noisy.
 
-For further details refer to the this [Falco documentation page](falco_ka_docs).
+For further details refer to the this [Falco documentation page][falco_ka_docs].
```

Comment:
Since you're doing fixes, how about:
```suggestion
For further details, see [Kubernetes Audit Events][falco_ka_docs] in the Falco documentation.
```

(`the this` is definitely not right in the original).

Reply:
BTW the changes
/lgtm

this feedback is about optional extra things to fix.